parts/k8s/manifests/kubernetesmaster-cloud-controller-manager.yaml

```diff
@@ -13,6 +13,13 @@ spec:
     - name: cloud-controller-manager
       image: {{ContainerImage "cloud-controller-manager"}}
       imagePullPolicy: IfNotPresent
+      {{- if IsAzureStackCloud}}
+      env:
+      - name: AZURE_ENVIRONMENT_FILEPATH
+        value: /etc/kubernetes/azurestackcloud.json
+      - name: AZURE_GO_SDK_LOG_LEVEL
+        value: DEBUG
```

Comment:
Do we want to keep this DEBUG verbosity for customers?

Reply:
I will update, thanks for catching this